### PR TITLE
Add configuration file for prettier

### PR DESCRIPTION
**Pull request by devmailar**
Just noticed that the project didn't have configuration for codeformating. This change could help other developers who maybe collaborate using prettier

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (agjs) [inline .prettierrc:1]:
```suggestion
{
    "trailingComma": "es5",
    "tabWidth": 4,
    "semi": true,
    "singleQuote": false
}
```

Comment (agjs) [reply .prettierrc:1]: Make sure you add this config, and then we can merge.

**Review by agjs — CHANGES_REQUESTED** [review]

File: .prettierrc [hunk 2]
```diff
@@ -1,4 +1,9 @@
 {
+    "trailingComma": "es5",
```
Comment (agjs) [inline .prettierrc:2]: @devmailar You can see that your commit got messed up :8). Try fixing it and we'll merge.

**Review by agjs — CHANGES_REQUESTED** [review]

File: .prettierrc (hunk 2 above)
Comment (devmailar) [reply .prettierrc:2]: Ok I fixed it

**Review by agjs — APPROVED** [review]

**Pull request merged**